Comp2024/src/main/java/frc/robot/Constants.java

```diff
@@ -296,22 +296,22 @@ public enum VIRequests
     ));
 
     //Poses for limelight paths
-    public static final Pose2d       kSpeakerPose   = new Pose2d(1.4, 5.52, Rotation2d.fromDegrees(180));
-    public static final Pose2d       kAmpPose       = new Pose2d(1.93, 7.31, Rotation2d.fromDegrees(90));
-    public static final Pose2d       kStageCenter   = new Pose2d(6.33, 4.13, Rotation2d.fromDegrees(180));
-    public static final Pose2d       kStageLeft     = new Pose2d(4.15, 5.52, Rotation2d.fromDegrees(-60));
-    public static final Pose2d       kStageRight    = new Pose2d(4.18, 2.67, Rotation2d.fromDegrees(60));
+    public static final Pose2d       kSpeakerPose   = new Pose2d(2.63, 4.03, Rotation2d.fromDegrees(0));
+    public static final Pose2d       kAmpPose       = new Pose2d(1.93, 7.31, Rotation2d.fromDegrees(-90));
+    public static final Pose2d       kStageCenter   = new Pose2d(5.92, 4.13, Rotation2d.fromDegrees(0));
+    public static final Pose2d       kStageLeft     = new Pose2d(4.3, 5.0, Rotation2d.fromDegrees(240));
+    public static final Pose2d       kStageRight    = new Pose2d(4.3, 3.2, Rotation2d.fromDegrees(-120));
 
     /////////////////////////////////////////////////////////////////////////////
     // Path on the fly trajectory constraints
     /////////////////////////////////////////////////////////////////////////////
     public static final class PATHConsts
     {
-      public static final double kMaxVelocityMps                         = 3.0;
-      public static final double kMaxAccelerationMpsSq                   = 3.0;
+      public static final double kMaxVelocityMps                         = 1.0;

```

Comment:
Please don't forget that these reductions in max speed, acceleration and angular values are temporary reductions. A TODO usually helps identify these.